tests/foreman/ui/test_contentview.py

```diff
@@ -611,11 +652,53 @@ def test_positive_add_rh_custom_spin(self):
 
         @assert: Filtered RH content only is available/can be seen in a view
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: Integration
         """
+        cv_name = gen_string('alpha')
+        filter_name = gen_string('alpha')
+        rh_repo = {
+            'name': REPOS['rhst7']['name'],
+            'product': PRDS['rhel'],
+            'reposet': REPOSET['rhst7'],
+            'basearch': 'x86_64',
+            'releasever': None
+        }
+        # Create new org to import manifest
+        org = entities.Organization().create()
+        with Session(self.browser) as session:
+            self.setup_to_create_cv(rh_repo=rh_repo, org_id=org.id)
+            # Create content view
+            make_contentview(session, org=org.name, name=cv_name)
+            self.assertIsNotNone(self.content_views.search(cv_name))
+            self.content_views.add_remove_repos(cv_name, [rh_repo['name']])
+            self.assertIsNotNone(self.content_views.wait_until_element(
+                common_locators['alert.success_sub_form']))
+            self.content_views.add_filter(
+                cv_name,
+                filter_name,
+                FILTER_CONTENT_TYPE['erratum by date and type'],
+                FILTER_TYPE['exclude']
+            )
+            # The last executed function will create the filter and open the
+            # edit filter form.
+            # will not assert the filter existence here to not exist from the
+            # filter edit form.
+            # will edit the filter fields immediately with open_filter=False
+            self.content_views.edit_erratum_date_range_filter(
+                cv_name,
+                filter_name,
+                errata_types=[FILTER_ERRATA_TYPE['enhancement'],
+                              FILTER_ERRATA_TYPE['bugfix']],
+                date_type=FILTER_ERRATA_DATE['issued'],
+                start_date='2016-01-01',
+                end_date='2016-06-01',
+                open_filter=False
```

Comment:
So... Do edit and add test methods has only one difference in whether we opening filter page or not? Not sure that I understand that, because editing and adding functionalities seem as pretty different things in logical way